plugins/plugin-wrk/src/lib/chart.ts

```diff
@@ -229,51 +210,5 @@ export const init = (graphics, options: IOptions = new DefaultOptions()) => {
   label.innerText = 'max rps'
   label.title = 'Maximum requests per second achieved during this run'
 
-  let currentMax
-  const updateMax = rps => {
-    if (!currentMax || rps > currentMax) {
-      max.innerText = (~~rps).toString()
-      currentMax = rps
-    }
-  }
-
-  if (!options.noListen) {
-    //
-    // register as a listener for load test updates, for the chart
-    //
-    const listener = eventBus.on('/wrk/iter', row => {
-      const { N: nThreads, requestsPerSec, /* rpsMin, rpsMax, */ latency25, latency50, latency75, latency90, latency99, latencyMax, latencyMin } = row
-
-      try {
-        const N = labels.length - 1
-        // labels[N] = new Date().getTime()
-        // l50[N] = parseDuration(latency50)
-        // l99[N] = parseDuration(latency99)
-        // lmax[N] = parseDuration(latencyMax)
-        // rps[N] = requestsPerSec
-        labels.push(row.timestamp)
-        lmin.data.push(parseDuration(latencyMin))
-        l25.data.push(parseDuration(latency25))
-        l50.data.push(parseDuration(latency50))
-        l75.data.push(parseDuration(latency75))
-        l99.data.push(parseDuration(latency99))
-        lmax.data.push(parseDuration(latencyMax))
-        rps.data.push(requestsPerSec)
-        // rpsmin.push(rpsMin * nThreads)
-        // rpsmax.push(rpsMax * nThreads)
-
-        // filler(10000)
-
-        updateMax(requestsPerSec)
-
-        // if (N > 0) {
-        chart.update()
-        // }
-      } catch (err) {
-        console.error(err)
-      }
-    })
-  }
-
```

Comment:
why did you remove this whole block of code?

Reply:
Readded

Reply:
it still shows up as removed for me

Reply:
re-requesting changes

Reply:
re-readded xD